src/extension/index.ts

```diff
@@ -48,6 +53,68 @@ export async function activate(context: ExtensionContext) {
     const panel = new Panel(context, baser, store);
     disposables.push(commands.registerCommand('sarif.showPanel', () => panel.show()));
 
+    // URI handler
+    disposables.push(window.registerUriHandler({
+        async handleUri(uri: vscode.Uri) {
+            if (uri.path === '/alert') {
+                // Launched by Azure DevOps Advanced Security alert page.
+                const pairs = uri.query.split('&');
```

Comment:
Consider getting a strongly-typed object via `new URLSearchParams(uri.query)`, which does all the splitting work for you.